### PR TITLE
Rename subTitle slot in the Expansion Panel samples

### LINKED ISSUE #265: Rename subTitle slot in the Expansion Panel samples
The 'subTitle' slot should be renamed to 'subtitle' in all expansion panel samples

### PR DESCRIPTION
**Pull request by MonikaKirkova**
Closes #265

### CODE REVIEW

**Review by mmart1n — APPROVED** [review]

File: samples/layouts/expansion-panel/properties-and-events/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <div class="panel-wrapper">
           <igc-expansion-panel id="exp-panel">
             <div slot="title">Golden Retriever</div>
-            <div slot="subTitle">Medium-large gun dog</div>
+            <div slot="subtitle">Medium-large gun dog</div>
             <div>
               <p>The Golden Retriever is a medium-large gun dog that retrieves shot waterfowl, such as ducks
               and upland game birds, during hunting and shooting parties.[3] The name "retriever" refers to the breed's ability
```
Comment (simeonoff) [inline samples/layouts/expansion-panel/properties-and-events/index.html:20]: Use `h1` for the title slot. Use `h3` for the subtitle slot. Do not wrap any of the content text in a div or paragraph - simply pass it as a text node.

File: samples/layouts/expansion-panel/properties-and-events/src/ExpansionPanelPropertiesAndEvents.css [hunk 2]
```diff
@@ -4,10 +4,14 @@
     height: 300px;
 }
 
+#exp-panel{
+    width: 100%;
+}
```
Comment (simeonoff) [inline samples/layouts/expansion-panel/properties-and-events/src/ExpansionPanelPropertiesAndEvents.css:9]: This shouldn't be needed. You can simply target the expansion panel by tag selector `igc-expansion-panel`. Also, I am not sure why setting the width to a 100% is explicitly needed.

**Review by simeonoff — CHANGES_REQUESTED** [review]
Use h1 for the title slot. Use h3 for the subtitle slot. Do not wrap any of the content text in a div or paragraph - simply pass it as a text node.

File: samples/layouts/expansion-panel/properties-and-events/src/ExpansionPanelPropertiesAndEvents.css (hunk 2 above)
Comment (MonikaKirkova) [reply samples/layouts/expansion-panel/properties-and-events/src/ExpansionPanelPropertiesAndEvents.css:9]: When the width of the expansion panel is not set, the header size is being changed when the panel is expanded/collapsed

**Comment by didimmova** [comment]
@MonikaKirkova 

You don't have <p> elements anywhere in the templates, but you're styling it. Also, I don't think that the 'panel-wrapper' and 'example-content' styles are needed as well.  I would put margin-top on the buttons as well to separate it a little bit from the content. And finally - I don't see the indicator in any of the samples.

**Review by simeonoff — APPROVED** [review]

**Pull request merged**